tutorminio/plugin.py

```diff
@@ -93,3 +94,15 @@ def add_minio_hosts(
         tutor_hooks.Filters.ENV_PATCHES.add_item(
             (os.path.basename(path), patch_file.read())
         )
+
+
+def is_discovery_enabled() -> bool:
+    for plugin, plugin_info in plugins.iter_info():
+        if plugin == "discovery":
+            return True
+    return False
```

Comment:
does tutor core provide a method for this? Wondering if there is something in core that can be leveraged instead of writing something altogether.

Reply:
I have updated the func using plugins.is_loaded() which is more suitable.